app/src/main/java/com/codesoom/assignment/domain/CatToyRegistory.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.domain;
+
+import com.codesoom.assignment.dto.CatToyNotFoundException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CatToyRegistory {
```

Comment:
혹시 `Repository`라고 쓰려고 하신 걸까요?
```suggestion
public class CatToyRepository {
```

Reply:
아 잘못 작성했네요. 감사합니다.